gravity-forms/gw-require-unique-values.php

```diff
@@ -153,7 +185,10 @@ public function get_value_hash( $value ) {
 
 	public function is_applicable_field( $field ) {
 
-		if ( ! $this->_args['field_ids'] ) {
+		// A single list field can be used for validation (validation logic to use unique values check over columns).
+		if ( ! $this->_args['field_ids'] && $field->get_input_type() == 'list' ) {
```

Comment:
The user should still be required to pass the List field ID as the `field_ids` parameter. The way this is written will make this apply to all List fields if no field IDs are passed. For example, this configuration shouldn't validate anything:

```
new GW_Require_Unique_Values( array(
	'form_id'   => 592,
	'field_ids' => array(),
) );
```

...but it will currently validate all List fields.

Reply:
Got it. Updated the condition here. Also, add a check on the constructor side to completely skip the validation if the field id (coming via the args) is empty. Thanks, @spivurno!